dist/index.d.ts

```diff
@@ -1977,10 +1977,6 @@ interface GPUAdapter {
    * </div>
    */
   readonly info: GPUAdapterInfo;
-  /**
-   * Returns the value of {@link GPUAdapter#[[adapter]]}.{@link adapter#[[fallback]]}.
-   */
-  readonly isFallbackAdapter: boolean;
```

Comment:
We should keep this as `@deprecated`, at least until Chrome Stable has the new `isFallbackAdapter`, but also make it `?` optional since we're planning to remove it.

Reply:
Done